### DISCUSSION TITLE
求教一个关于\item的问题

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#161] [Q&A]
你好，论文一章，编译的时候报错：
`Something's wrong--perhaps a missing \item.`
但是把每一段分开运行都是正常的，剔除任何一段也是正常的，就是所有段落一起运行反而出现这个错误。这会是什么原因造成的，特别奇怪。

**Comment by xkwxdyy** [comment]
这个报错原因有很多，你这提供的信息无法做任何诊断

**Reply by earthmoon** [reply]
> 这个报错原因有很多，你这提供的信息无法做任何诊断

方便远程吗，如果我把一章放上来，查重都有风险哎。

**Reply by tanukihee** [reply]
建议使用无意义的文字代替内容

**Reply by earthmoon** [reply]
> 建议使用无意义的文字代替内容

用1代替任何一段，这个问题也消失了。就很奇怪

**Reply by tanukihee** [reply]
一个字一个字，一个字母一个字母，一个标点一个标点地换

**Reply by earthmoon** [reply]
> 一个字一个字，一个字母一个字母，一个标点一个标点地换

几十页的文档，删除任何一段问题消失，会不会是bug。

**Reply by tanukihee** [reply]
请以 **字** 而 **不是** **段** 为单位进行 debug

**Reply by earthmoon** [reply]
> 请以 **字** 而 **不是** **段** 为单位进行 debug

可以远程吗，这么多字，一个字一个字搞不现实啊。

**Reply by earthmoon** [reply]
有一个特征，报错之前，会卡一下。

**Reply by tanukihee** [reply]
不足以构成有用的信息，加油吧

**Reply by earthmoon** [reply]
> 不足以构成有用的信息，加油吧

搞不懂这个问题，把其中一段缩小内容也可以，应该是内部处理排版的问题。谢谢你了。

**Reply by earthmoon** [reply]
看了网上的信息，其他人也遇到存在`\enumerate`都是正常的但是报错`\item`。

**Reply by syvshc** [reply]
修改 `enumerate` 环境前的任何一个字都会导致 bug 消失？

**Reply by earthmoon** [reply]
> 修改 `enumerate` 环境前的任何一个字都会导致 bug 消失？

不会，删掉前面两行文字就可以，是纵向的间距造成的。

**Comment by earthmoon** [comment]
此问题由`afterpage + list = missing \item`造成，目前没有完美解决方案，`afterpage`是一个弱命令，慎重使用。
